cli/script/command-executor.ts

```diff
@@ -657,7 +776,38 @@ function printDeploymentHistory(command: cli.IDeploymentHistoryCommand, packageH
                     releaseTime,
                     packageObject.appVersion,
                     packageObject.isMandatory ? "Yes" : "No",
-                    packageObject.description ? wordwrap(30)(packageObject.description) : ""
+                    packageObject.description ? wordwrap(30)(packageObject.description) : "",
+                    getPackageMetricsString(packageObject)
+                ]);
+            });
+        });
+    }
+}
+
+function printDeploymentMetrics(command: cli.IDeploymentMetricsCommand, metrics: DeploymentMetrics): void {
+    var labelRegex = /^v\d+$/;
+    var totalActive: number = 0;
+    Object.keys(metrics).forEach((label: string) => {
+        totalActive += metrics[label].active;
+    });
+    
+    if (command.format === "json") {
+        printJson(metrics);
+    } else if (command.format === "table") {
+        printTable(["Label", "Type", "Active", "Downloaded", "Installed", "Failed"], (dataSource: any[]) => {
```

Comment:
Should we show downloads here? I'd prefer to be consistent with the other commands and display the pending install count instead.

Additionally, instead of adding the "Type" column, what if we added the "App version" column,  and for binary releases, the "Label" column would be empty. If you made the "App version" column the first column, it would act as a sort of "section header" for all of the CodePush releases it contains.